src/lib/libidbfs.js

```diff
@@ -13,13 +13,10 @@ addToLibrary({
   $IDBFS: {
     dbs: {},
     indexedDB: () => {
-      if (typeof indexedDB != 'undefined') return indexedDB;
-      var ret = null;
-      if (typeof window == 'object') ret = window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB;
 #if ASSERTIONS
-      assert(ret, 'IDBFS used, but indexedDB not supported');
+      assert(typeof indexedDB != 'undefined', 'IDBFS used, but indexedDB not supported');
 #endif
-      return ret;
+      return (typeof indexedDB != 'undefined') ? indexedDB : null;
```

Comment:
Since this function is allowed to throw if it's missing anyway (in assertions mode at least), might as well just do `return indexedDB;`.